include/whole_body_state_rviz_plugin/WholeBodyStateDisplay.h

```diff
@@ -1,14 +1,18 @@
 ///////////////////////////////////////////////////////////////////////////////
 // BSD 3-Clause License
 //
-// Copyright (C) 2020, University of Edinburgh, Istituto Italiano di Tecnologia
+// Copyright (C) 2020-2021, University of Edinburgh, Istituto Italiano di
+// Tecnologia, University of Oxford.
 // Copyright note valid unless otherwise stated in individual files.
 // All rights reserved.
 ///////////////////////////////////////////////////////////////////////////////
 
 #ifndef WHOLE_BODY_STATE_RVIZ_PLUGIN_WHOLE_BODY_STATE_DISPLAY_H
 #define WHOLE_BODY_STATE_RVIZ_PLUGIN_WHOLE_BODY_STATE_DISPLAY_H
 
+#include "whole_body_state_rviz_plugin/ArrowVisual.h"
```

Comment:
Do we need to move these header before pinocchio stuffs?

I often follow include the internal headers after the global ones.

Reply:
The formatting did the re-order as far as I can recall